### PR TITLE
update kodi version and hash

### PR DESCRIPTION
**Pull request by FinalDes**

### CODE REVIEW

**Comment by vitorgalvao** [comment]
Please submit separate pull requests for each cask, as more often than not this type of pull request brings problems with one or more of them, and it halts the inclusion of the other ones.

In this case, there a few things that need to be clarified/corrected in the new addition.

File: Casks/osmc.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask :v1 => 'osmc' do
+  version :latest
+  sha256 :no_check 
+
```
Comment (vitorgalvao) [inline Casks/osmc.rb]: Please remove one of these empty lines.

File: Casks/osmc.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  
+  url 'http://download.osmc.tv/installers/osmc-installer.dmg'
+  name 'OSMC'
+  homepage 'https://osmc.tv/'
+  license :gpl
+  
+  app 'qt_host_installer.app', :target => 'OSMC_setup.app'
```
Comment (vitorgalvao) [inline Casks/osmc.rb]: If this is an installer, then it needs [`installer :manual`](https://github.com/caskroom/homebrew-cask/blob/master/doc/CASK_LANGUAGE_REFERENCE.md#installer-manual), not `app`. That wold also means it’d need [`uninstall`](https://github.com/phinze/homebrew-cask/blob/master/doc/CASK_LANGUAGE_REFERENCE.md#uninstall-stanza-details).

Comment (FinalDes) [inline Casks/osmc.rb]: this is a the app install burn the firmware into sd card

Comment (vitorgalvao) [inline Casks/osmc.rb]: It’s a bit hard to explain the changes, so I’ll pull your PR and make them.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It will be merged in [a different pull request](https://github.com/caskroom/homebrew-cask/pull/9436) due to some simple recommended alterations. Your contribution is still included (and still credited to you), with the appropriate modifications, when applicable.

One of the changes is related to `:target`, so here’s a screenshot of how the app looks when open, for future reference.

**Pull request closed without merging**